### PR TITLE
Update German translations 

### PR DESCRIPTION
**Pull request by imbstt**
There was some discussion in my circles on Bluesky, that the recently-added translation of "liked" → "likt" (and its variations) looked weird and uncommon. 

As far as I can tell, this originated from the last update by @surfdude29 using DeepL, although it is not *wrong*, but there's also no 100% fixed consensus about it in German grammar literature. I personally think "liked" works better than "likt" in German. 

While I was at it, I modified and added a bunch more translations, although I don't have time to complete any more currently.

### CODE REVIEW

File: src/locale/locales/de/messages.po [hunk 1]
```diff
@@ -100,11 +100,11 @@ msgstr "Konto stummgeschaltet"
 #: src/components/moderation/ModerationDetailsDialog.tsx:93
 #: src/lib/moderation/useModerationCauseDescription.ts:91
 msgid "Account Muted"
-msgstr "Konto Stummgeschaltet"
+msgstr "Konto stummgeschaltet"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:103]: Our initial thought here was that the English translation is uppercased, hence we did the same for the German translation. I guess that's open for debate? I'm fine with both versions.

File: src/locale/locales/de/messages.po [hunk 2]
```diff
@@ -270,7 +270,7 @@ msgstr "Eine E-Mail wurde an deine vorherige Adresse {0} gesendet. Sie enthält
 
 #: src/lib/moderation/useReportOptions.ts:26
 msgid "An issue not included in these options"
-msgstr ""
+msgstr "Ein Problem, das hier nicht aufgelistet ist."
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:273]:
```suggestion
msgstr "Ein Problem, das hier nicht aufgelistet ist"
```

I'm not sure what the context of this translation is, but the English translation has no `.` at the end of the sentence, so maybe the actual sentence continues with something else?

File: src/locale/locales/de/messages.po [hunk 3]
```diff
@@ -1350,27 +1350,27 @@ msgstr "Aufgrund der Apple-Richtlinien können Inhalte für Erwachsene erst nach
...
 
 #: src/lib/moderation/useGlobalLabelStrings.ts:43
 msgid "E.g. artistic nudes."
-msgstr ""
+msgstr "Z.B. künstlerische Nacktheit"
 
 #: src/view/com/modals/CreateOrEditList.tsx:284
 msgid "e.g. Great Posters"
-msgstr "z.B. Große Poster"
+msgstr "z.B. Großartige Poster"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:1373]: Yeah, I guess that's what was meant? 😁

File: src/locale/locales/de/messages.po [hunk 4]
```diff
@@ -1588,28 +1588,28 @@ msgstr "Alle"
...
 
 #: src/view/com/modals/DeleteAccount.tsx:230
 msgid "Exits account deletion process"
-msgstr ""
+msgstr "Verlässt den Vorgang der Accountlöschung"
 
 #: src/view/com/modals/ChangeHandle.tsx:151
 msgid "Exits handle change process"
-msgstr "Beendet den Prozess des Handle-Wechsels"
+msgstr "Verlässt den Vorgang des Handle-Wechsels"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:1599]: 👍

File: src/locale/locales/de/messages.po [hunk 5]
```diff
@@ -2380,35 +2380,35 @@ msgstr "Diesen Feed liken"
 #: src/Navigation.tsx:201
 #: src/Navigation.tsx:206
 msgid "Liked by"
-msgstr "Gelikt von"
+msgstr "Geliked von"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:2383]: 🎉

**Review by PythooonUser — COMMENTED** [review]
Looks good! Thanks for taking care!

File: src/locale/locales/de/messages.po (hunk 1 above)
Comment (imbstt) [reply src/locale/locales/de/messages.po:103]: I understood it as some sort of title for a dialog/modal, but titles aren't all uppercased in German like they often are in English (see e.g. Wikipedia article structures)

Comment (PythooonUser) [reply src/locale/locales/de/messages.po:103]: Sure, I'm fine with your approach

**Review by pfrazee — APPROVED** [review]
Thanks all!!

**Pull request merged**